app/src/main/java/com/jerboa/datatypes/Others.kt

```diff
@@ -121,6 +122,19 @@ fun getLocalizedStringForUserTab(
         UserTab.Comments -> ctx.getString(R.string.person_profile_screen_comments)
     }
 
+/**
+ * Returns localized Strings for BlocksTab Enum
+ */
+fun getLocalizedStringForBlocksTab(
+    ctx: Context,
+    tab: BlocksTab,
```

Comment:
Make the R.string a property of the enum.

Don't need a switch then

Reply:
Done, thanks for the tip